aztk/spark/models.py

```diff
@@ -123,9 +123,9 @@ def __init__(
             application=None,
             application_args=None,
             main_class=None,
-            jars=[],
-            py_files=[],
-            files=[],
+            jars=None,
```

Comment:
don't you need to do something like `self.jars = jars or []` instead to keep the same

Reply:
That's what line [141](https://github.com/Azure/aztk/pull/392/files#diff-73e8698c9fc8e05c277a1850e41dac89R141) does unless I'm misunderstanding what you're saying.

Reply:
oh nvm missed that, didn't saw it for the ones above but it actually check for not null so it doesn't matter